resources/admin/classes/domain/Resource.php

```diff
@@ -1366,7 +1366,7 @@ public function getAlphabeticalList() {
 		$alphArray = array();
 		$result = $this->db->query("SELECT DISTINCT UPPER(SUBSTR(TRIM(LEADING 'The ' FROM titleText),1,1)) letter, COUNT(SUBSTR(TRIM(LEADING 'The ' FROM titleText),1,1)) letter_count
 								FROM Resource R
-								GROUP BY SUBSTR(TRIM(LEADING 'The ' FROM titleText),1,1)
+								GROUP BY SUBSTR(TRIM(LEADING 'The ' FROM titleText),1,1), titleText
```

Comment:
Your solution worked on my instance.

I would go for any solution, but I'm not sure which one is the best, performance-wise (or if they are equivalent)